### ISSUE TITLE
Node not appearing

### ISSUE THREAD

**Issue by tristan22mc** [brayevalerien/ComfyUI-resynthesizer#1]
Im using comfyui portable do I need to do the pip install resynthesizer command after installing via github link?

**Comment by tristan22mc** [comment]
Update I installed package into my python environment using the command: s\ComfyUI_windows_portable>.\python_embeded\python.exe -m pip install --target=".\python_embeded\Lib\site-packages" resynthesizer

And now it is saying the node has installed successfully on my startup logs however when I go to add the node via searching for the node and clicking it, it does not appear. Also the test workflow still shows the node missing (red)

**Comment by brayevalerien** [comment]
Hi @tristan22mc, thank you for the interest you are showing to this node.

The installation process is poorly documented, I'll try to work on it tomorrow, but I just tried to install this node on another computer and I had no issues. Note that I'm not using ComfyUI standalone version but I actually have a global Python install and use Anaconda to manage environments. I would advise against using the standalone version if you can, and since I don't know exactly how it's structured I can't ensure it works with it, but there shoudn't be any difference.

 here is what I did:
1. Activate my Conda environment: `conda activate comfyui` (this is probably not needed for the standalone install, I don't know if it uses a venv?)
2. Clone this repo inside my `ComfyUI/custom_nodes` directory: `git clone https://github.com/brayevalerien/ComfyUI-resynthesizer`. 
3. Move to the cloned directory: `cd ComfyUI-resynthesizer`
4. Install `resynthesizer`: `pip install resynthesizer -I` (`-I` is just to force re-install if I had the package already installed fsr)
5. Launch ComfyUI

Here I could add the node and use it as intended, even with the little change I did in the last commit (fd75f5c). 

**In conclusion:** I could not reproduce your issue from a fresh install, maybe uninstall everything and redo it. If it still is not showing, check for any error in the terminal when ComfyUI is starting and tell me so I can work on a fix. In the mean time I'll work on streamlining the installation process.

---

**Quick note**: this extension has been made available in the ComfyUI Manager, so you can also get it from here. **But I don't recommend it as of now** since I did not expected that to happen and forgot to add a requirements file. If you still want to try to get it from the manager though, you will still need to run `pip install resynthesizer` (maybe in a virtual environment if ComfyUI standalone uses one).

**Comment by dmytro-khyzhniak** [comment]
I have faced the same issue when:
* Using ComfyUI portable
* Installing your custom node from ComfyUI Manager

Error `Node fails to import with the following: ModuleNotFoundError: No module named 'resynthesizer'` is thrown

It can be resolved by installing `requirements.txt` from `ComfyUI_windows_portable\python_embeded`.
